storage/skip_localtest.go

```diff
@@ -0,0 +1,12 @@
+package storage
+
+import (
+	"os"
+	"testing"
+)
+
+func SkipLocalTest(t *testing.T) {
+	if os.Getenv("CI") == "" {
```

Comment:
Is this symbol added CI environment?

Reply:
Thank you for your review!
I requested the DevOps to register the CI environment variable.
And I modified the ci variable value.